src/librustc_middle/ty/context.rs

```diff
@@ -36,7 +36,9 @@ use rustc_data_structures::sync::{self, Lock, Lrc, WorkerLocal};
 use rustc_errors::ErrorReported;
 use rustc_hir as hir;
 use rustc_hir::def::{DefKind, Res};
-use rustc_hir::def_id::{CrateNum, DefId, DefIdMap, LocalDefId, LOCAL_CRATE};
+use rustc_hir::def_id::{
+    CrateNum, DefId, DefIdMap, LocalDefId, RawDefId, CRATE_DEF_INDEX, LOCAL_CRATE,
+};
```

Comment:
Could you avoid multi-line imports?

Reply:
I'll change this, but I think it would be a good idea to (eventually) have `rustfmt` enforce the preferred import style.